edn/src/symbols.rs

```diff
@@ -161,8 +223,9 @@ impl Display for NamespacedKeyword {
     /// ```rust
     /// # use edn::symbols::NamespacedKeyword;
     /// assert_eq!(":bar/baz", NamespacedKeyword::new("bar", "baz").to_string());
+    /// assert_eq!(":bar/_baz", NamespacedKeyword::new("bar", "_baz").to_string());
```

Comment:
Also add a `.to_reversed()` test here. You could make this new assertion pass without changing any of the other code in the file!